README.md

```diff
@@ -148,6 +148,14 @@ class Account < ApplicationRecord
 end
 ```
 
+Custom index can be handled with a `Arel::Nodes::SqlLiteral` instance:
+
+```ruby
+class Account < ApplicationRecord
+  upsert_keys Arel::Nodes::SqlLiteral.new('md5(my_long_field)')
+end
```

Comment:
When using the `Arel::Nodes::SqlLiteral` way, are there other kinds of structure that you want to be able to pass in here, OR would it be less unsafe to use some sort of interface here, such as:

```ruby
upsert_keys { literal: 'md5(my_long_field)' }
```

...which would later be converted to the `Arel::Nodes::SqlLiteral`.

I'm curious about how much Arel is nice to use.